dev/reference/tools/data-migration/troubleshoot/dm.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Data Migration Troubleshooting
+summary: Learn how to diagnose and resolve issues when you use Data Migration.
+category: reference
+aliases: ['/docs/dev/how-to/troubleshoot/data-migration/']
+---
+
+# Data Migration Troubleshooting
+
+This document summarizes some commonly encountered issues when you use Data Migration, and provides the solutions.
```

Comment:
this document doesn't contain issues, only tell users how to check DM and then find the problem